lib/utils/remote-build.ts

```diff
@@ -42,6 +42,7 @@ export interface BuildOpts {
 	headless: boolean;
 	convertEol: boolean;
 	multiDockerignore: boolean;
+	isdraft: boolean;
```

Comment:
```suggestion
	isDraft: boolean;
```
... to follow the camelCase convention for variable / function / class names, even if the query string argument had to read `isdraft` because perhaps `balena-builder` requires it (does it? Another query string argument reads `dockerfilePath`, with camelCase).

Reply:
balena-builder requires it like this. @toochevere how do you feel about changing your PR to use camelCase for the `isDraft` query param?

Reply:
@pdcastro I did change other instances in this PR though